.github/workflows/main.yml

```diff
@@ -74,18 +74,18 @@ jobs:
 
   linux-avatica:
     if: github.event.action != 'labeled'
-    name: 'Linux (JDK 11), Avatica master'
+    name: 'Linux (JDK 11), Avatica-1.19.0'
```

Comment:
Do you suggest each upgrade of Avatica would trigger manual replacement of 1.19.0 in several places?
I am afraid it creates a really sad maintainer experience.